### PR TITLE
Implement Interface views in speedy

### PR DESCRIPTION
**Pull request by dylant-da**

### CODE REVIEW

**Review by remyhaemmerle-da — APPROVED** [review]
Looks good to me. Thanks.

**Comment by remyhaemmerle-da** [comment]
cc @cocreature @nmarton-da

File: daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/ValueEnricher.scala [hunk 1]
```diff
@@ -65,14 +65,10 @@ final class ValueEnricher(
   ): Result[Value] = for {
     // TODO: https://github.com/digital-asset/daml/issues/14112
```
Comment (cocreature) [inline daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/ValueEnricher.scala:66]: You can remove this todo and maybe even close the issue if the other todos & stuff in there are also addressed by this.

**Review by cocreature — APPROVED** [review]
Nice, should we add a test that calls this somewhere? Currently you only have the tests for computeInterfaceView but I can also call this from Daml and testing that seems useful.

File: daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/ValueEnricher.scala (hunk 1 above)
Comment (dylant-da) [reply daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/ValueEnricher.scala:66]: This todo is done, but some require small tweaks before I can get rid of them
Making another PR for the small fixes necessary to eliminate the remaining todos

**Comment by dylant-da** [comment]
> should we add a test that calls this somewhere?

I'll put this into another PR

**Pull request merged**